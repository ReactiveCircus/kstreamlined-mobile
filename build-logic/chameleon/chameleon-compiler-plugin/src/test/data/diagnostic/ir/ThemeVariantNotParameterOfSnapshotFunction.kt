// SNAPSHOT_FUNCTION: /SnapshotTester2.snapshot
// THEME_VARIANT_ENUM: /ThemeVariant2
// CHECK_COMPILER_OUTPUT

class SnapshotTester2 {
    fun snapshot(
        content: @Composable () -> Unit,
    ) {}
}

enum class ThemeVariant2 {
    Light,
    Dark,
}
