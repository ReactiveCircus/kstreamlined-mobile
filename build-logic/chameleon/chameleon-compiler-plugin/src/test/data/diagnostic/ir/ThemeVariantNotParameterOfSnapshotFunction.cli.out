error: theme variant enum </ThemeVariant2> is not a parameter of snapshot function </SnapshotTester2.snapshot>.
